sirepo/package_data/static/js/raydata.js

```diff
@@ -725,13 +821,47 @@ SIREPO.app.directive('scanSelector', function() {
 		);
 	    };
 
+	    $scope.setColumns = function() {
+		$scope.columns = ['select'].concat($scope.defaultColumns.concat(appState.models.metadataColumns.selected));
+	    };
+
+	    $scope.setAvailableColumns = function() {
+		$scope.availableColumns = $scope.masterListColumns.filter(function(value) {
+		    return !$scope.columns.includes(value) && !$scope.defaultColumns.includes(value);
```

Comment:
Something to think about - it would be nice if we could reference the concept of defaultColumns less (possibly only when updating appState).

Reply:
e64969d8a3a9303ebcf97566971db0411515341f